config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   devise_for :users
 
+  resources :spaces
```

Comment:
looks like you added all possible routes ! cool !

Reply:
Ok can you merge ?

Reply:
believe in yourself. Let's do it

Reply:
Je suis d'accord

Reply:
well done guys Im proud of you